collectionfunctions.php

```diff
@@ -53,17 +54,16 @@ function getPokes(PDO $database): Array {
  * @return array
  */
 function getPoke(PDO $database): Array {
-    $getPokemon = $database->prepare("SELECT `stats`.`name`, `stats`.`type1`, `stats`.`type2`, `stats`.`hp`, `stats`.`attack`, `stats`.`defense`, `stats`.`spAttack`, `stats`.`spDefense`, `stats`.`speed` FROM `stats` WHERE `id` = '{$_GET['id']}' AND `deleted` = 0;");
-    $getPokemon->execute();
-    return $getPokemon->fetch();
+    $getPoke = $database->prepare("SELECT `stats`.`name`, `ty`.`type` AS `type1`, `ty2`.`type` AS `type2`, `stats`.`hp`, `stats`.`attack`, `stats`.`defense`, `stats`.`spAttack`, `stats`.`spDefense`, `stats`.`speed` FROM `stats` LEFT JOIN `types` `ty` ON `stats`.`type1` = `ty`.`id` LEFT JOIN `types` `ty2` ON `stats`.`type2` = `ty2`.`id` WHERE `stats`.`id` = :id AND `deleted` = 0;");
+    $getPoke->execute(['id' => $_GET['id']]);
+    return $getPoke->fetch();
 }
 
 /** Links page to style sheets
  * @return String
  */
 function addStyle(): String {
-    $input = '';
-    $input .= "<link rel='preconnect' href='https://fonts.googleapis.com' />";
+    $input = "<link rel='preconnect' href='https://fonts.googleapis.com' />";
     $input .= "<link rel='preconnect' href='https://fonts.gstatic.com' crossorigin />";
     $input .= "<link href='https://fonts.googleapis.com/css2?family=Nunito&display=swap' rel='stylesheet' />";
     $input .= "<link rel='stylesheet' href='normalize.css' />";
```

Comment:
As lame as it may sound, this should actually be unit tested 😂 

I missed that before so don't worry about it!